src/cclib/io/filewriter.py

```diff
@@ -53,6 +52,10 @@ def __init__(self, ccdata, jobfilename=None, terse=False,
             self.obmol, self.pbmol = self._make_openbabel_from_ccdata()
             self.bond_connectivities = self._make_bond_connectivity_from_openbabel(self.obmol)
 
+        # Check if required attributes are present.
```

Comment:
You can also set `required_attrs` in the base class to the empty list, and then you don't need to do this check.